web_src/js/features/codeeditor.ts

```diff
@@ -114,8 +146,11 @@ export async function createMonaco(textarea: HTMLTextAreaElement, filename: stri
   ]);
 
   const model = editor.getModel();
-  model.onDidChangeContent(() => {
-    textarea.value = editor.getValue({preserveBOM: true});
+  model?.onDidChangeContent(() => {
```

Comment:
When it would be `null`?

Reply:
I don't know, but the type signature suggests it can be null:

https://microsoft.github.io/monaco-editor/typedoc/functions/editor.getModel.html

Reply:
I'm not going to dig into vscode sources now for this, but I'd happily put `!` which will restore previous behaviour because `!` has no runtime effect while `?` does.